README.md

```diff
@@ -82,9 +82,12 @@ theme:
 # - privacy: Enable location services
 # - timezone: Set the timezone
 # - telemetry: Enable sending telemetry
+#
+# Do note that only some non-mandatory pages offer the possibility to be hidden.
```

Comment:
Should we name the pages that can be hidden explicitly?

Reply:
Right, I forgot to circle back on that, which ones do we want to be able to hide?
Are accessibility, source, try-or-install and refresh enough for stage 3?